src/redux/books/books.js

```diff
@@ -0,0 +1,31 @@
+// Write action creators to add and remove books
+export const addBook = (book) => ({
+  type: 'BOOK_ADDED',
+  title: book.title,
+  author: book.author,
+  id: book.id,
+});
+export const removeBook = (book) => ({
+  type: 'BOOK_REMOVED',
+  id: book.id,
+});
+
+// Create reducer for booklist
+const bookReducer = (state = [], action) => {
+  switch (action.type) {
+    case 'BOOK_ADDED':
+      return [
+        ...state,
+        {
+          title: action.title,
+          author: action.author,
+          id: action.id,
+        },
+      ];
+    case 'BOOK_REMOVED':
+      return state.map((id) => id !== action.id);
```

Comment:
- [ ] For the remove book action creator, we should use ES6 filter() method which returns a new array without a book we want to remove